pkg/process/util/status.go

```diff
@@ -66,29 +66,31 @@ type MemInfo struct {
 
 // ProcessExpvars holds values fetched from the exp var server
 type ProcessExpvars struct {
-	Pid                 int                 `json:"pid"`
-	Uptime              int                 `json:"uptime"`
-	UptimeNano          float64             `json:"uptime_nano"`
-	MemStats            MemInfo             `json:"memstats"`
-	Version             InfoVersion         `json:"version"`
-	DockerSocket        string              `json:"docker_socket"`
-	LastCollectTime     string              `json:"last_collect_time"`
-	ProcessCount        int                 `json:"process_count"`
-	ContainerCount      int                 `json:"container_count"`
-	ProcessQueueSize    int                 `json:"process_queue_size"`
-	RTProcessQueueSize  int                 `json:"rtprocess_queue_size"`
-	EventQueueSize      int                 `json:"event_queue_size"`
-	PodQueueSize        int                 `json:"pod_queue_size"`
-	ProcessQueueBytes   int                 `json:"process_queue_bytes"`
-	RTProcessQueueBytes int                 `json:"rtprocess_queue_bytes"`
-	EventQueueBytes     int                 `json:"event_queue_bytes"`
-	PodQueueBytes       int                 `json:"pod_queue_bytes"`
-	ContainerID         string              `json:"container_id"`
-	ProxyURL            string              `json:"proxy_url"`
-	LogFile             string              `json:"log_file"`
-	EnabledChecks       []string            `json:"enabled_checks"`
-	Endpoints           map[string][]string `json:"endpoints"`
-	DropCheckPayloads   []string            `json:"drop_check_payloads"`
+	Pid                   int                 `json:"pid"`
+	Uptime                int                 `json:"uptime"`
+	UptimeNano            float64             `json:"uptime_nano"`
+	MemStats              MemInfo             `json:"memstats"`
+	Version               InfoVersion         `json:"version"`
+	DockerSocket          string              `json:"docker_socket"`
+	LastCollectTime       string              `json:"last_collect_time"`
+	ProcessCount          int                 `json:"process_count"`
+	ContainerCount        int                 `json:"container_count"`
+	ProcessQueueSize      int                 `json:"process_queue_size"`
+	RTProcessQueueSize    int                 `json:"rtprocess_queue_size"`
+	ConnectionsQueueSize  int                 `json:"connections_queue_size"`
```

Comment:
We may want to add the new fields to the templated used to render the status command:
https://github.com/DataDog/datadog-agent/blob/1d15a135240a86d31a48e86e0e983c73b1cbf24c/pkg/status/templates/process-agent.tmpl#L39-L54

Reply:
Updated, thank you!